bagitobjecttransfer/recordtransfer/static/recordtransfer/css/base/modal.css

```diff
@@ -0,0 +1,89 @@
+.modal {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5);
+    z-index: 1000;
+    visibility: hidden;
+    opacity: 0;
+    padding: 0;
+}
+
+.modal.visible {
+    visibility: visible;
+    opacity: 1;
+}
+
+.modal-content {
+    background: white;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+}
+
+.modal-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+    border-bottom: 1px solid #ddd;
+}
+
+.modal-header h3 {
+    margin: 0;
+    flex-grow: 1;
+    text-align: left;
+}
+
+.close-modal-button {
+    background: none;
+    border: none;
+    cursor: pointer;
+}
+
+.close-modal-button img {
+    display: block;
+}
+
+.modal-body {
+    padding: 10px;
+    text-align: left;
+}
+
+.modal-footer {
+    display: flex;
+    justify-content: flex-end;
+    padding-top: 10px;
+}
+
+.modal-footer button {
+    margin-left: 15px;
+}
+
+/* Responsive styles for screens below 800px */
+@media (max-width: 800px) {
+    .modal-content {
+        max-width: 80%;
+        padding: 15px;
+        font-size: small;
+    }
+
```

Comment:
The font is a little too small when the screen is under 800px wide. Without the `font-size` rule the modal size is still manageable.

```suggestion
    .modal-content {
        max-width: 80%;
        padding: 15px;
    }
```

This is without `font-size: small;` (i.e., with my change incorporated):

![Screenshot 2025-01-05 165325](https://github.com/user-attachments/assets/e62dd1d2-beec-4844-9486-c1e4f593fa8e)

Reply:
I agree, I've made the change here https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/288/commits/47ba9410b3930de358c26d921b193f4f61174f67